scss/_card.scss

```diff
@@ -47,6 +47,7 @@
   // as much space as possible, ensuring footers are aligned to the bottom.
   flex: 1 1 auto;
   padding: $card-spacer-y $card-spacer-x;
+  padding-bottom: $spacer + 8;
```

Comment:
Not comprehending this change. Seems like it would make a headerless, footerless card look asymmetric.